bitnami/redis/templates/networkpolicy.yaml

```diff
@@ -52,11 +52,15 @@ spec:
               {{ template "common.names.fullname" . }}-client: "true"
         - podSelector:
             matchLabels: {{- include "common.labels.matchLabels" . | nindent 14 }}
-        {{- if .Values.networkPolicy.ingressNSMatchLabels }}
+        {{- if or .Values.networkPolicy.ingressNSMatchLabels .Values.networkPolicy.ingressNSPodMatchLabels }}
         - namespaceSelector:
             matchLabels:
-              {{- range $key, $value := .Values.networkPolicy.ingressNSMatchLabels }}
-              {{ $key | quote }}: {{ $value | quote }}
+              {{- if or .Values.networkPolicy.ingressNSMatchLabels }}
```

Comment:
Here there is a `or` directive but only one value, could you please check it?

Reply:
Good catch -- fixed.